lib/spack/spack/test/concretize.py

```diff
@@ -348,31 +348,33 @@ def test_compiler_flags_differ_identical_compilers(self):
             assert spec.satisfies("cflags=-O2")
 
     def test_concretize_compiler_flag_propagate(self):
-        spec = Spec("hypre cflags=='-g' ^openblas")
+        spec = Spec("callpath cflags=='-g'")
```

Comment:
Can we add new tests rather than changing old ones? Or is there a reason for changing them?